cli/src/cli.rs

```diff
@@ -2241,7 +2254,7 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
                 .arg(
                     Arg::with_name("address_signer")
                         .index(2)
-                        .value_name("ADDRESS_SIGNER")
+                        .value_name("PROGRAM_ADDRESS_SIGNER")
```

Comment:
This is referenced in the deploy failure ephemeral keypair recovery message.  Mind updating there too?